modules/src/ics03_connection/connection.rs

```diff
@@ -31,7 +31,7 @@ impl Default for ConnectionEnd {
             client_id: Default::default(),
             counterparty: Default::default(),
             versions: vec![],
-            delay_period: Duration::from_secs(0),
+            delay_period: Duration::default(),
```

Comment:
Nit: I'm not sure about this. Makes sense that `Duration::default()` would be a 0, but I have a slight preference for being explicit with the default value (unless it's a type we define ourselves, such as `ClientId`).

Reply:
Agreed, I too would prefer to be more explicit.

Reply:
Too bad [`Duration::ZERO`](https://doc.rust-lang.org/stable/std/time/struct.Duration.html#associatedconstant.ZERO) is nightly-only for now.

Reply:
Yeah `Duration::ZERO` is only in nightly. I have added a constant `ZERO_DURATION` in `ibc::timestamp` and use that instead.

My main issue with `Duration::from_secs(0)` is that it obscures the other use of `Duration::from_secs` during searching, and there can be potential inconsistency of using `Duration::from_secs(0)` and `Duration::from_nanos(0)` in different places.